conductr_cli/sandbox_run_jvm.py

```diff
@@ -466,7 +467,7 @@ def extract_binary(path, conductr_info):
 def download_sandbox_image(image_dir, package_name, artefact_type, image_version):
     try:
         bintray_auth = bintray_resolver.load_bintray_credentials(raise_error=False)
-
+        bintray_repo = BINTRAY_CONDUCTR_COMMERCIAL_REPO if bintray_auth[0] else BINTRAY_CONDUCTR_GENERIC_REPO
```

Comment:
Perform the check based on the image version. We already have a function in `sandbox_version` that you can use: `is_conductr_on_private_bintray`